easybuild/easyblocks/e/easybuildmeta.py

```diff
@@ -65,6 +66,18 @@ def __init__(self, *args, **kwargs):
             # consider setuptools first, in case it is listed as a sources
             self.easybuild_pkgs.insert(0, 'setuptools')
 
+    # Override this function since we want to respect the user choice for the python installation to use
+    # (which can be influenced by EB_PYTHON and EB_INSTALLPYTHON)
+    def prepare_python(self):
+        """Python-specific preparations."""
+
+        self.python_cmd = sys.executable
+        self.log.info("Python command being used: %s", self.python_cmd)
+
+        if self.python_cmd:
```

Comment:
Not sure it's worth doing this conditionally, can `sys.executable` ever be `None`?

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2581/commits/d8c7d4ea6c24e942c03b56291c601c51bbe750b6